modules/axhal/src/arch/loongarch64/context.rs

```diff
@@ -0,0 +1,286 @@
+use core::arch::naked_asm;
+use memory_addr::VirtAddr;
+/// Saved registers when a trap (interrupt or exception) occurs.
+#[repr(C)]
+#[derive(Debug, Default, Clone, Copy)]
+pub struct TrapFrame {
+    /// All general registers.
+    pub regs: [usize; 32],
+    /// Pre-exception Mode Information
+    pub prmd: usize,
+    /// Exception Return Address
+    pub era: usize,
+    /// Access Memory Address When Exception
+    pub badv: usize,
```

Comment:
`badv` and `crmd` don't need to be saved in `TrapFrame`, do they? We can use `csrrd` to get them when we need them.